gradle.properties

```diff
@@ -13,6 +13,7 @@ android.enableJetifier=true
 android.useAndroidX=true
 org.gradle.jvmargs=-Xmx1536m
 
+
```

Comment:
Redundant change.
Ideally, this file should not be in VCS as it is a setting that would change from system to system. Can you please check in which PR (and see any specific reason to add this file) this file was added.